components/omega/doc/design/OmegaV1GoverningEqns.md

```diff
@@ -0,0 +1,1589 @@
+
+$$
+& \frac{d}{dt} \int_{A} \int_{z^{\text{bot}}}^{z^{\text{top}}} \rho \, {\bf u} \, dz \, dA
++
+   \int_{A}\nabla_\perp \cdot \left( \int_{z^{\text{bot}}}^{z^{\text{top}}}\rho\, {\bf u} \otimes {\bf u} \, dz \right) dA
+ + \int_{A}\left[ \rho\, {\bf u} \left(w - w_r \right) \right]_{z=z^{\text{top}}} \, dA
+ - \int_{A}\left[ \rho\, {\bf u} \left(w - w_r \right) \right]_{z=z^{\text{bot}}} \, dA
+\\ & \; =
+- \int_{A} \int_{z^{\text{bot}}}^{z^{\text{top}}} \rho \,  \mathbf{f} \times \mathbf{u} \, dz \, dA
+- \int_{A} \int_{z^{\text{bot}}}^{z^{\text{top}}} \rho \, \nabla_\perp \Phi \, dz \, dA
+ - \int_{A} \int_{z^{\text{bot}}}^{z^{\text{top}}} 
+  \nabla_{\perp} p \, dz \, dA
++ \int_{A} \int_{z^{\text{bot}}}^{z^{\text{top}}} 
+ \frac{\partial}{\partial x_i} \left( \tau_{ij} \right) 
+\, dz \, dA.
+$$ (h-momentum-p-tau)
+
+Dividing by $\rho_0$ and taking vertical averages using [](#def-thickness) and [](#def-pseudo-thickness),
+
+$$
+\frac{d}{dt} \int_{A} \tilde{h}\, \overline{  {\bf u} }^{\tilde z} \, dA
++
+   \int_{A}\nabla_\perp \cdot \left( \tilde{h}\, \overline{{\bf u} \otimes {\bf u} }^{\tilde z} \right) dA
+& + \int_{A}\left[ {\bf u}\, \tilde{w}_{tr} \right]_{z=z^{\text{top}}} \, dA
+ - \int_{A}\left[ {\bf u}\, \tilde{w}_{tr} \right]_{z=z^{\text{bot}}} \, dA
+\\ & \; =
+- \int_{A} \tilde{h} \,\overline{  \mathbf{f} \times \mathbf{u} }^{\tilde z} \, dA
+- \int_{A} \tilde{h} \,\overline{  \nabla_\perp \Phi }^{\tilde z} \, dA
+- \int_{A} \frac{1}{\rho_0} \overline{\nabla_{\perp} p}^{z}  \, dA
++ \int_{A} \frac{1}{\rho_0} \overline{ \frac{\partial}{\partial x_i} \left( \tau_{ij} \right) }^z dA
+$$ (h-momentum-p-tau)
+
+Taking the limit as $A \rightarrow 0$, we arrive at the local, horizontally continuous form:
+
+$$
+\frac{\partial \tilde{h}\, \overline{  {\bf u} }^{\tilde z} }{\partial t} 
+& +
+   \nabla_\perp \cdot \left( \tilde{h}\, \overline{{\bf u} \otimes {\bf u} }^{\tilde z} \right) 
+ + \left[ {\bf u}\, \tilde{w}_{tr} \right]_{z=z^{\text{top}}} 
+ - \left[ {\bf u}\, \tilde{w}_{tr} \right]_{z=z^{\text{bot}}} 
+\\ & =
+-  \tilde{h} \,\overline{  \mathbf{f} \times \mathbf{u} }^{\tilde z} 
+-  \tilde{h} \,\overline{  \nabla_\perp \Phi }^{\tilde z} 
+-  \frac{1}{\rho_0} \overline{\nabla_{\perp} p}^{z}  
++  \frac{1}{\rho_0} \overline{ \frac{\partial}{\partial x_i} \left( \tau_{ij} \right) }^z 
+$$ (momentum-layered-differential-1)
+
+The first two terms are the material derivative, confined within the horizontal layer. Using the product rule, and layered mass conservation [](#layer-mass),
+
+$$
+\frac{\partial \tilde{h}\, \overline{  {\bf u} }^{\tilde z} }{\partial t} 
++ \nabla_\perp \cdot \left( \tilde{h}\, \overline{{\bf u} \otimes {\bf u} }^{\tilde z} \right) 
+&= 
+\frac{D_\perp \tilde{h}\, \overline{  {\bf u} }^{\tilde z} }{D t} \\
+&=
+\tilde{h}\frac{D_\perp  \overline{  {\bf u} }^{\tilde z} }{D t} 
++
+\overline{ {\bf u} }^{\tilde z} \frac{D_\perp \tilde{h}}{D t} \\
```

Comment:
The right-hand side seems to be missing the vertical velocity terms to make it the full material derivative.